packages/video-ad-sdk/src/adContainer/__tests__/VideoAdContainer.spec.js

```diff
@@ -53,6 +56,7 @@ describe('VideoAdContainer', () => {
     expect(videoAdContainer.videoElement.parentNode).toBe(adContainerElement);
   });
 
+  // eslint-disable-next-line jest/no-disabled-tests
```

Comment:
Can this be removed now? It doesn't look like the test is disabled.

Reply:
👍